shared/yarn.lock

```diff
@@ -13,9 +13,9 @@
   version "1.1.8"
   resolved "https://registry.yarnpkg.com/@ant-design/icons/-/icons-1.1.8.tgz#a95dbe1e32928500ab130fb35f77e2ca447ec0aa"
 
-"@babel/cli@7.1.0":
-  version "7.1.0"
-  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.1.0.tgz#a9429fd63911711b0fa93ae50d73beee6c42aef8"
+"@babel/cli@^7.0.0":
```

Comment:
You should use `yarn modules` instead of `yarn install`. It's an alias for `yarn install --pure-lockfile`. And please rebase.

Reply:
Cool, let me do that, Thanks for the info